apps/strapi-cms/src/api/webinar/content-types/webinar/lifecycles.ts

```diff
@@ -0,0 +1,41 @@
+// The only way to make this code work is to do side effects, and throwing exceptions and creating functions
+/*eslint-disable*/
+import { errors } from '@strapi/utils';
+
+interface IWebinarEvent {
+  readonly params: {
+    readonly data: {
+      readonly startDatetime?: string;
+      readonly endDatetime?: string;
+    };
+  };
+}
+
+const validateDates = (event: IWebinarEvent) => {
+  const { data } = event.params;
+
+  const startDateTime = data.startDatetime
+    ? new Date(data.startDatetime)
+    : null;
+  const endDateTime = data.endDatetime ? new Date(data.endDatetime) : null;
+
+  if ((startDateTime && !endDateTime) || (!startDateTime && endDateTime)) {
```

Comment:
Wouldn't it be simpler like this:
```suggestion
  if (!startDateTime || !endDateTime) {
```

Reply:
No, with your correction the validation would be triggered even if both values are falsey
With `startDate = null, endDate = null`, in this case the validation should not be triggered
but with your correction the if will evaluate true on the first condition and therefore execute the body of the if.

Reply:
**suggestion**: Add unit tests

based on the discussion, I think it would be good to add unit tests 😄

Reply:
It's on strapi side, we currently do not have tests for strapi internals. We could set them up, but maybe it's better to create its own task for this activity because i think will not be trivial